### DISCUSSION TITLE
NetTopologySuite with DynamoDb and entityFrameworkCore

### DISCUSSION THREAD

**Discussion by cissemy** [NetTopologySuite/NetTopologySuite#559] [Q&A]
Hi
How can I use NetTopologySuite with Dynamodb and entityFrameworkCore ?
Thanks

**Comment by bricelam** [comment]
You'd need an [EF Core provider](https://docs.microsoft.com/ef/core/providers/) for Amazon DynamoDB first...

**Comment by bricelam** [comment]
Currently, EF Core with NTS is only [available on](https://docs.microsoft.com/ef/core/modeling/spatial#installing)...

* Microsoft SQL Server
* PostgreSQL (with PostGIS)
* SQLite (with SpatiaLite)
* MySQL
* Teradata
